### PR TITLE
(GH-43) Use Svg or Png for icons

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

**Comment by gep13** [comment]
@ferventcoder is this enough? Or do we need to be smarter about discerning the file type?

File: src/chocolatey.package.validator/infrastructure.app/rules/UsePngOrSvgForIconSuggestion.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    using NuGet;
+    using infrastructure.rules;
+
+    public class UsePngOrSvgForIconSuggestion : BasePackageRule
+    {
+        public override string ValidationFailureMessage
+        {
+            get
+            {
+                return "As per the [packaging guidelines]() icons should be either a png or svg file.  Please see [here](https://github.com/chocolatey/package-validator/wiki/UsePngOrSvgForPackageIcons) for further information and guidance.";
```
Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/UsePngOrSvgForIconSuggestion.cs]: Limit use of "here" as link text - fails accessibility standards.

File: src/chocolatey.package.validator/infrastructure.app/rules/UsePngOrSvgForIconSuggestion.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            }
+        }
+
+        protected override PackageValidationOutput is_valid(IPackage package)
+        {
+            return package.IconUrl.ToString().EndsWith(".png") || package.IconUrl.ToString().EndsWith(".svg");
```
Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/UsePngOrSvgForIconSuggestion.cs]: Use only .to_string(). The one you've used is not null safe.

**Comment by gep13** [comment]
@ferventcoder updated with your suggestions.

File: src/chocolatey.package.validator/infrastructure.app/rules/UsePngOrSvgForIconSuggestion.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    using NuGet;
+    using infrastructure.rules;
+
+    public class UsePngOrSvgForIconSuggestion : BasePackageRule
+    {
+        public override string ValidationFailureMessage
+        {
+            get
+            {
+                return "As per the [packaging guidelines]() icons should be either a png or svg file.  Please see [wiki article](https://github.com/chocolatey/package-validator/wiki/UsePngOrSvgForPackageIcons) for further information and guidance.";
```
Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/UsePngOrSvgForIconSuggestion.cs]: Might want a link here.

Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/UsePngOrSvgForIconSuggestion.cs]: `[packaging guidelines]() `

Comment (gep13) [inline src/chocolatey.package.validator/infrastructure.app/rules/UsePngOrSvgForIconSuggestion.cs]: @ferventcoder doh! good catch! :smile: 

Fixed it up.

**Pull request merged**